<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iPartVariants</class>
 <widget class="QWidget" name="iPartVariants">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle"><string>iPart</string></property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnPartAdd">
       <property name="text">
        <string>Add Variant</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPartDel">
       <property name="text">
        <string>Delete Variant</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnParamAdd">
       <property name="text">
        <string>Add Parameter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnParamDel">
       <property name="text">
        <string>Delete Parameter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
